src/lib/viewers/controls/page/PageControls.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import IconArrowDown24 from '../icons/IconArrowDown24';
+import IconArrowUp24 from '../icons/IconArrowUp24';
+import PageControlsForm from './PageControlsForm';
+import './PageControls.scss';
+
+export type Props = {
+    onPageChange: (newPageNumber: number) => void;
+    onPageSubmit: (newPageNumber: number) => void;
+    pageCount: number;
+    pageNumber: number;
+};
+
+export default function PageControls({ onPageChange, onPageSubmit, pageCount, pageNumber }: Props): JSX.Element {
+    return (
+        <div className="bp-PageControls">
+            <button
+                className="bp-PageControls-button"
+                data-testid="bp-PageControls-previous"
+                disabled={pageNumber === 1}
```

Comment:
Should this be <= 1 since the prop passed in can be any number?

Reply:
True!  I can make it <=1.  Ideally it should never go below 0 since the viewer prevents it, but <=1 is a better check.